less/index.less

```diff
@@ -50,10 +131,278 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html,
+body {
+	height: 100%;
+}
+
+@defaultfontsize: 62.5%;
+@mobilefontsize: 100%;
+
+@mobilebreakpoint: 500px;
+@tabletbreakpoint: 750px;
+@desktopbreakpoint: 1000px;
+
+// @xlargefontsize: 2rem;
+// @largefontsize: 1.5rem;
+// @mediumfontsize: 1rem;
+// @smallfontsize: 0.5rem;
+
+// general styles
+:root {
+	font-size: @mobilefontsize;
+}
+section {
+	margin-bottom: 20px;
+}
+.container {
+	flex-flow: column;
+	margin: 0 auto;
+	// max-width: @desktopbreakpoint;
+	@media (min-width: @tabletbreakpoint) {
+		flex-direction: row;
+		background-color: yellow;
+	}
+	@media (min-width: @desktopbreakpoint) {
+		max-width: @desktopbreakpoint;
+		flex-direction: row;
+		background-color: pink;
+	}
+}
+
+nav {
+	display: flex;
+	flex-direction: column;
+
+	@media (min-width: @tabletbreakpoint) {
+		flex-direction: row;
+		justify-content: flex-end;
+	}
+	@media (min-width: @desktopbreakpoint) {
+		flex-direction: row;
+		background-color: pink;
+	}
+}
+nav a {
+	display: flex;
+	justify-content: center;
+	border-bottom: 1px solid grey;
+	padding: 10px 0;
+	margin: 0;
+	color: grey;
+	&:hover {
+		background-color: rgb(99, 107, 214);
+		color: white;
+	}
+	@media (min-width: @tabletbreakpoint) {
+		width: 10%;
+		justify-content: flex-end;
+		border: none;
+		&:hover {
+			background-color: pink;
+		}
+		@media (min-width: @desktopbreakpoint) {
+			width: 10%;
+			justify-content: flex-end;
+			border: none;
+			&:hover {
+				background-color: pink;
+			}
+		}
+	}
```

Comment:
Be careful with your nesting the second breakpoint is currently nested inside the second. Also if they are the same styling you can just use the larger of the two breakpoints